template/package.json

```diff
@@ -40,7 +41,11 @@
     "metro-react-native-babel-preset": "^0.66.2",
     "patch-package": "^6.4.7",
     "postinstall-postinstall": "^2.1.0",
+    "react-native-fast-image": "8.5.11",
+    "react-native-interactable": "0.1.10",
+    "react-native-linear-gradient": "2.5.6",
```

Comment:
At first I was able to avoid having to re-add these to the template. My logic for keeping it was that the projects final product requires no dependencies and all of these are for "development" or testing